ActionSheet.android.js

```diff
@@ -83,10 +83,15 @@ class ActionGroup extends React.Component {
       false
     );
 
+    let textStyle = [styles.text];
+
     for (let i = startIndex; i < startIndex + length; i++) {
       let color = '#444444';
       if (i === destructiveButtonIndex) {
         color = '#ff3b30';
+        textStyle = textStyle.concat([ this.props.textStyle, {color} ])
```

Comment:
please fix the coding style to match the rest of this file for spacing, semis, etc. 

also please revert this change, it's a breaking change, and keep separate changes in separate PRs

Reply:
I create new one cherry-picking only the work of @kesha-antonov on prop-types fix : https://github.com/expo/react-native-action-sheet/pull/59